test/serializers/cache_test.rb

```diff
@@ -34,6 +34,22 @@ def setup
         @blog_serializer     = BlogSerializer.new(@blog)
       end
 
+      def test_inherited_cache_configuration
+        inherited_serializer = Class.new(PostSerializer)
+
+        assert_equal PostSerializer._cache_key, inherited_serializer._cache_key
+        assert_equal PostSerializer._cache_options, inherited_serializer._cache_options
+      end
+
+      def test_override_cache_configuration
+        inherited_serializer = Class.new(PostSerializer) do
+          cache key: 'new-key'
+        end
+
+        assert PostSerializer._cache_key == 'post'
+        assert inherited_serializer._cache_key == 'new-key'
```

Comment:
Awesome! Please change to `assert_equal` and I think this is good to go.

Do you think there's any value in testing all the class_attributes?  I think there would be
```ruby
     serializer.class_attribute :_cache
      serializer.class_attribute :_fragmented
      serializer.class_attribute :_cache_key
      serializer.class_attribute :_cache_only
      serializer.class_attribute :_cache_except
      serializer.class_attribute :_cache_options
      serializer.class_attribute :_cache_digest
```

I have some similar changes in https://github.com/bf4/active_model_serializers/commit/a60ba9dff8a02877a828657d4308778080dee7fb#diff-5624f9cf35be4dc254412d979fd293c4R31 that could go along with or follow this:

```diff

+    class_attribute :_attributes, instance_writer: false
+    self._attributes ||= []
+    class_attribute :_attributes_keys, instance_writer: false, instance_reader: false
+    self._attributes_keys ||= {}
+    class_attribute :_type, instance_writer: false
#
   def self.inherited(base)
+      base._attributes = _attributes.dup
+      base._attributes_keys = _attributes_keys.dup
#
    def attributes
+      attributes = _attributes.dup

      attributes.each_with_object({}) do |name, hash|
+        if _fragmented
+          hash[name] = _fragmented.public_send(name)
+        else
+          hash[name] = send(name)
        end
      end
    end
```